iot_config/src/gateway_service.rs

```diff
@@ -184,14 +184,23 @@ impl iot_config::Gateway for GatewayService {
         })?;
 
         let (region, gain) = match self.resolve_gateway_info(address).await {
-            Err(_) => {
+            Err(status) if status.code() == tonic::Code::NotFound => {
                 tracing::debug!(
                     pubkey = %address,
                     %default_region,
                     "unable to retrieve gateway from chain"
                 );
                 (default_region, 0)
             }
+            Err(status) => {
+                tracing::error!(
+                    pubkey = %address,
+                    %default_region,
+                    "gateway lookup failed"
+                );
+                telemetry::count_region_lookup(default_region, Region::Unknown);
```

Comment:
why count this as a region unknown?

Reply:
i didn't want to not count the request in the event we had unexpected errors and the failure count suddenly spiked and it seemed technically correct that we couldn't in fact verify the region. also i didn't want to make the metric count take an option for this probably unlikely scenario but i could do that as well

Reply:
would it then not bucket the failed lookups with the unknown region lookups? i.e. it'd miscount unasserted DO hotspots?

Reply:
Yeah, thinking more about it that second argument should be an option

Reply:
fixed [d93d2b6](https://github.com/helium/oracles/pull/565/commits/d93d2b64a7f2bad7e20f19d220e8adf5c07b2af4)